tests/huawei_vrp/display_bgp_peer/huawei_vrp_display_bgp_peer.yml

```
@@ -0,0 +1,38 @@
+---
+parsed_sample:
+  - as: "635.66"
+    msg_rcv: "47622"
+    msg_sent: "49953"
+    out_q: "0"
+    peer: "192.168.1.11"
+    pref_rcv: "551"
+    state: "Established"
+    up_down: "626h43m"
+    v: "4"
+  - as: "635.66"
+    msg_rcv: "47036"
+    msg_sent: "40065"
+    out_q: "0"
+    peer: "192.168.1.12"
+    pref_rcv: "551"
+    state: "Established"
+    up_down: "626h43m"
+    v: "4"
+  - as: "635.66"
+    msg_rcv: "47795"
+    msg_sent: "40010"
+    out_q: "0"
+    peer: "192.168.1.13"
+    pref_rcv: "551"
+    state: "Established"
+    up_down: "626h43m"
+    v: "4"
+  - as: "635.66"
+    msg_rcv: "47293"
+    msg_sent: "49912"
+    out_q: "0"
+    peer: "192.168.1.14"
+    pref_rcv: "551"
```

And a similar structured data update, this time for `pref_rcvdD`

```suggestion
    pref_rcvd: "551"
    state: "Established"
    up_down: "626h43m"
    v: "4"
  - as: "635.66"
    msg_rcv: "47036"
    msg_sent: "40065"
    out_q: "0"
    peer: "192.168.1.12"
    pref_rcvd: "551"
    state: "Established"
    up_down: "626h43m"
    v: "4"
  - as: "635.66"
    msg_rcv: "47795"
    msg_sent: "40010"
    out_q: "0"
    peer: "192.168.1.13"
    pref_rcvd: "551"
    state: "Established"
    up_down: "626h43m"
    v: "4"
  - as: "635.66"
    msg_rcv: "47293"
    msg_sent: "49912"
    out_q: "0"
    peer: "192.168.1.14"
    pref_rcvd: "551"
```